README.md

```diff
@@ -148,7 +148,7 @@ For security, you need to enter something into the `SECRET_KEY`.
 SECRET_KEY = 'somethingnotempty'
 ```
 
-By default, you need a `Memcache` server up and running. The connection
+*By default, you need a `Memcache` server up and running.* The connection
```

Comment:
Keep adding commits until you get an r+ comment. Then, when you have one of those, go ahead and rebase your PR down to 1 neat messaged commit and force push that into your branch and once the travis tests have tested one last time, I merge.